src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/TypecastParenPadCheck.java

```diff
@@ -30,7 +30,7 @@
  * </p>
  * <ul>
  * <li>
- * Property {@code option} - Specify policy on how to pad parentheses.
+ * Property {@code option} - Set the option to enforce.
```

Comment:
Why we need to change description ?

Reply:
```
diff (-expected +actual):
    @@ -4,7 +4,7 @@
     </p>
     <ul>
     <li>
    -Property {@code option} - Set the option to enforce. Type is {@code com.puppycrawl.tools.checkstyle.checks.whitespace.PadOption}. Default value is {@code nospace}.
    +Property {@code option} - Specify policy on how to pad parentheses Type is {@code com.puppycrawl.tools.checkstyle.checks.whitespace.PadOption}. Default value is {@code nospace}.
     </li>
     </ul>
     <p>
```

`Set the option to enforce.` is extracted from `AbstractParenPadCheck` which is parent of `TypecastParenPadCheck`.

Reply:
https://github.com/checkstyle/checkstyle/blob/63b9e0b71097538b0a1ed59d32108f923cf934d1/src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/AbstractParenPadCheck.java#L73-L79

Reply:
https://checkstyle.org/checks/whitespace/typecastparenpad.html#Properties

> option | Specify policy on how to pad parentheses.

https://checkstyle.org/checks/whitespace/parenpad.html#Properties

> option | Specify policy on how to pad parentheses.


Lets change wording of abstract class setter.